### PR TITLE
Aligning target size SCs

### LINKED ISSUE #3086: Further alignment of structure/wording of 2.5.5 Target Size (Enhanced) and 2.5.8 Target Size (Minimum)
There's currently already an issue about aligning the "Inline" definition https://github.com/w3c/wcag/issues/2857 but I'd suggest going further and making sure the structure and wording of the two SCs are not unnecessarily different.

* 2.5.8 Target Size (Minimum) introduces the exceptions with "except *where*:" and then has an *unordered list* for the exceptions; 2.5.5 Target Size (Enhanced) introduces the exceptions with "except *when*:" and then uses a *description list* for the exceptions
* 2.5.8 has the following definition/exception: "Equivalent: The function can be achieved through a different control on the same page that meets this criterion."; 2.5.5 uses different wording to mean the same? "Equivalent: The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;"
* 2.5.8 has the following definition/exception: "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) or is legally required for the information being conveyed;"; 2.5.5 has "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) to the information being conveyed." ... so, is the "legally required" aspect of no concern for AAA ? And if so, why? Or just an oversight? (this leaves me with a weird "can't do AAA and be legal" feeling.

I'd say pick one of these as the authoritative wording and format, and stick with it consistently. And then harmonise the exceptions that are meant to be the same, and use the same consistent wording for both the AA and AAA version

### PR DESCRIPTION
**Pull request by alastc**
Closes #3086

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/pull/3097.html" title="Last updated on May 29, 2024, 2:38 PM UTC (f84e036)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/3097/f3e9113...f84e036.html" title="Last updated on May 29, 2024, 2:38 PM UTC (f84e036)">Diff</a>

### CODE REVIEW

**Comment by patrickhlauke** [comment]
> So is it intentional that you may potentially not be able to be legal and follow AAA?

**Comment by mbgower** [comment]
> [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) or is legally required for the information being conveyed

nice catch. My recollection was that we were being explicit adding "legal" and that some felt it was redundant. But in for a penny.... Should be the same for both, I think?

File: guidelines/sc/22/target-size-minimum.html [hunk 1]
```diff
@@ -6,13 +6,18 @@ <h4>Target Size (Minimum)</h4>
...
-        <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion.</li>
-        <li><strong>Inline:</strong> The target is in a sentence, or is in a bulleted or numbered list, or its size is otherwise constrained by the line-height of non-target text;</li>
-        <li><strong>User agent control:</strong> The size of the target is determined by the user agent and is not modified by the author;</li>
-        <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</li>
-    </ul>
+    <dl>
+        <dt>Spacing:</dt> 
+        <dd>The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</dd>
+        <dt>Equivalent:</dt> 
+        <dd>The function can be achieved through a different control on the same page that meets this criterion.</dd>
```
Comment (patrickhlauke) [inline guidelines/sc/22/target-size-minimum.html:13]: change `.` at the end of this to `;`

File: guidelines/sc/22/target-size-minimum.html [hunk 2]
```diff
@@ -6,13 +6,18 @@ <h4>Target Size (Minimum)</h4>
...
+        <dt>Inline:</dt> 
+        <dd>The target is in a sentence, or is in a bulleted or numbered list, or its size is otherwise constrained by the line-height of non-target text;</dd>
+        <dt>User agent control:</dt> 
+        <dd>The size of the target is determined by the user agent and is not modified by the author;</dd>
+        <dt>Essential:</dt> 
+        <dd>A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</dd>
```
Comment (patrickhlauke) [inline guidelines/sc/22/target-size-minimum.html:19]: change the `;` at the end of this to `.`

**Comment by patrickhlauke** [comment]
need to see how https://github.com/w3c/wcag/pull/3097 affects this

**Comment by patrickhlauke** [comment]
need to work out relationship with https://github.com/w3c/wcag/pull/3123

**Comment by patrickhlauke** [comment]
I would have actually liked this alignment of normative wording - but can't recall if we already have that again in another PR

**Comment by fstrr** [comment]
Re-opened and added to the project board as "in progress". Not sure if we have it in another PR—hopefully adding it to the board will generate a discussion around that.

**Comment by patrickhlauke** [comment]
Looking at this again, it *seems* that what this was doing (roughly anyway) was covered already by https://github.com/w3c/wcag/pull/3189 which is now waiting for CfC, so in hindsight it looks like it *can* be closed.

**Pull request closed without merging**